int8 servo_id
int16 position